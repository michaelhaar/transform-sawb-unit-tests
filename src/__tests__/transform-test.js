const defineSnapshotTestFromFixture = require('jscodeshift/dist/testUtils').defineSnapshotTestFromFixture;
const transform = require('../transform');
const transformOptions = {};

defineSnapshotTestFromFixture(__dirname, transform, transformOptions, 'sample-1', 'should work as expected');
defineSnapshotTestFromFixture(__dirname, transform, transformOptions, 'sample-2', 'should work as expected');
