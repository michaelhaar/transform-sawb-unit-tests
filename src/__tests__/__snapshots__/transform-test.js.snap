// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should work as expected 1`] = `
"const bar = 42;

const favoriteFruit = "apple";"
`;
